### ISSUE TITLE
[BUG]: There are no descriptive aria-labels for icon-only buttons that show on hover

### ISSUE THREAD

**Issue by silasrogan** [azukaar/Cosmos-Server#539]
### What happened?

When I hover over the ServApp card deployment icons (or indeed every icon-only button so far, except for the Server Terminal button) there are no aria-labels visible after a second as expected. Thus, I cannot meaningfully tell what the respective function is. Inspecting the element shows aria-labels for the icon name itself, rather than the function the icon performs, i.e. "close-square" rather than "Destroy Container."

### What should have happened?

When I hover over any icon-only button I should see a tooltip with the description text of what function the icon represents. I've attached the following edited image as an example.

<img width="1626" height="1102" alt="Image" src="https://github.com/user-attachments/assets/aa3b1c24-3b22-4ef4-8521-e07817679784" />

### How to reproduce the bug?

1. Go to ServApps
2. Hover over any icon-only button below the container heading such as the pause icon
3. Wait a few moments
4. No text appears

Successful example of an aria-label that works
1. Visit any page on Cosmos Cloud
2. Hover over the first icon-only button on the top right navigation bar
3. You will see "Server Terminal" appear after a few milliseconds.

### Relevant log output

```shell

```

### Other details

Would be great to have the text labels appear on hover since full text buttons wouldn't fit. My memory isn't great, so I rely on small interactions like this to quickly get reacquainted.

### System details

- OS: Ubuntu Server
- Browser: Arc Browser
- Version 0.22.9
